app/src/main/java/org/oppia/app/walkthrough/WalkthroughPageChanger.kt

```diff
@@ -1,7 +1,10 @@
 package org.oppia.app.walkthrough
 
-/** Listener for when an activity should change pages. */
+/** Listener for when an activity should change pages during the app walkthrough flow's welcome
+ * fragment.
+ */
```

Comment:
```suggestion
/**
 * Listener for when an activity should change pages during the app walkthrough flow's welcome
 * fragment.
 */
```

Notice the formatting--we always format KDocs like this unless they can fit on one line. See: https://developer.android.com/kotlin/style-guide#formatting_2.

Reply:
Okay, thank you!